openapi3/schema_validation_settings.go

```diff
@@ -65,3 +67,8 @@ func newSchemaValidationSettings(opts ...SchemaValidationOption) *schemaValidati
 	}
 	return settings
 }
+
+// SetOpenAPIMinorVersion setting allows to define minor OpenAPI version schema must comply with
+func SetOpenAPIMinorVersion(minorVersion uint64) SchemaValidationOption {
```

Comment:
This seems easier to read when used:
```suggestion
func SetOpenAPI(v float32) SchemaValidationOption {
```
And also panic if `NaN` or if not finite or actually just if not `3.0` or `3.1`.

Reply:
We are in the openapi3 package, so I assume this is for any OpenAPI 3.x definition. So having 3.0 is useless and we would have to ensure it is not <3.0 nor >= 4.0.

Using an uint also has the advantages of using 0 as default (not 3.0).